### DISCUSSION TITLE
buddhism/2024/mind_ground/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#117] [General]
# buddhism/latest/mind_ground/

https://shanenull.com/buddhism/latest/mind_ground/

**Comment by shane0** [comment]
# first letters

```text
codie
rfmc
baas
an
lg
aeimaiai
snaon
```

**Comment by shane0** [comment]
- add tts https://peakd.com/hive-163399/@peakd/listen-to-posts-with-the-new-text-to-speech-feature

**Reply by shane0** [reply]
https://peakd.com/buddhism/@shane0/mind-ground-chan-meditation

**Comment by shane0** [comment]
# spontaneous enlightenment

- no time is needed
- there is nothing to practice
- nothing to realize
- nothing to gain
- nothing to lose
- throughout all time there is no other Dharma than this
- https://youtube.com/clip/UgkxY6juFK7QLjd8R-N76uvN9FD7p0yKi7zV?si=tipqB6LvLuJ1vDJd

**Comment by shane0** [comment]
# mind ground meditation notes 

- 3 Chan meditation course from the level 3 book chapter 10 page 82

- prepare: repeat 3x "cessation of delusion is enlightenment"
- remain focused mindful confident
- be aware and still
- abide nowhere
- let go
- all experiences in meditation are illusory and insubstantial
- seek nothing and obtain nothing

## 1. Remain focused and mindful, and be confident

Collect the mind, do not allow it to wander around.
This method is free of any meditation object. Abiding without something to anchor the mind to may seem difficult, for our mind is so used to grasping or relying on something.
Nonetheless, we can and should build confidence in our ability to bring about the state of depending on nothing but our pure awareness - our knowing.
It's not so much about having self-confidence; it's more important to be confident that there's no self. Furthermore, we are the only, true master of our mind.
In other words, the true self is our inherently pure mind, which is free from any of the four problems we might encounter in meditation.

## 2. Be aware and still

Try to keep our mind fully aware and still, a mind of great lucidity, clarity, and stability.
Master Yongjia (665-712) taught: "The mind being aware and still is right; the mind being aware, yet restless is not right; the mind being still, yet blank is not right, either."
Therefore, in every moment, we try to maintain a mind of clarity, stillness, insight, and serenity.
Our mind has complete autonomy, undistracted in any condition, remaining watchful under any circumstances.
The very mind can sail effortlessly and naturally, and is at ease all the time.
The Grand Master once said, "Let the mind be as it is, intrinsically free; stay firm in the seal of the true Dharma... and
"Firmly abide in right mindfulness, and naturally follow our awareness."
Simply put, be aware and be still.

## 3. Abide nowhere

The Diamond Sutra teaches that bodhisattvas "should give rise to a pure mind that does not abide in form, sound, smell, taste, touch, or dharmas; they should abide in nothing to give rise to the mind."
We have learned several meditation methods involving meditation objects on which our mind can rely.
While meditating with the mind ground, we remain aware; in other words, just maintain our pure awareness, without contemplation or visualization. As stated before, we remain highly attentive, yet still. Be present, do not abide in anything, and keep our very mind in the state of pure knowing. As we prolong the time in this state of pure awareness, our afflictions will gradually subside. If we are perfectly aware-the master of our mind-for one minute, our Dharma body is present in the world for one minute. If we can meditate without a single thought for ten minutes, the true buddha within manifests for ten minutes.

**Reply by shane0** [reply]
# 4. Let go

Try to abide nowhere, not giving rise to a single thought, for as long as we can. However, as ordinary beings, it is very difficult to maintain a state of mind without thoughts. When thoughts bubble up, do not take any actions to stop them. Do not analyze the quality of our unwanted thoughts, do not investigate why they appear, and do not wonder where they come from. Self-denigration or self-criticism is not helpful either.
We can imagine that we are waiting at a level crossing for a train to pass. We sit quietly, patiently, and mindfully in our car.
Maintain this knowing and simply be attentive. We do not look at the train, do not think about where it is coming from, and do not ask where it is going to. When the train is passing by in front of our eyes, we simply know it is passing by; our eyes do not fall on the train and our eyesight does not follow it either.
At the same time, we are fully aware of the train coming and going, the noise it makes, and the sound of the warning signal.
When the train has passed, we also know it has passed, and we can safely drive through the level crossing.
When we see the red lights flashing, we know a train is approaching. When we see a train passing by right before our eyes, we know it is moving. When it has passed, and we no longer see the train in front of us, we know it has gone.
This analogizes the nature of our thoughts-rising, abiding, changing, and ceasing. No matter what type of train comes, how many passengers it has, how long it is, or how old it is, we do not care. Likewise, do not try to figure out the number of our thoughts, nor their qualities; just maintain our pure knowing, without any differentiation or judgment.
Again, the best antidote to deal with our thoughts is to hane no espons at al. Vien the we remain a et them o As our thoughts begin to subside, we vemain aware. When ou thoughts cease, we also know it. Yet, we do not give any of aus thoughts any special attention.
This approach does not suggest that we deny the existence of thoughts, nor that we pretend they do not exist. The principle ofletting go teaches us that the nature of thoughts is provisional and deceiving. A thought is provisional because it merely exist for a fleeting moment. Since it will disappear anyway, why do we waste our time confronting it? It is deceptive and elusive, because it is not what we really see. Since it is delusional, why do we try to reject or hold onto something inexistent?
Positive thoughts are like sparklers, beautiful but ephemeral.
Negative thoughts are like the dummies in a haunted house at a theme park, frightening but unreal. When we remain calm and unmoved, our thoughts have no power over us. When we maintain mindfulness and true knowing, we can deconstruct the falsehood of thoughts. Seeing through our thoughts, we are freed from them.
If we pay attention to a thought, we create more thoughts' All thoughts, arising and ceasing, symbolize the endless births and deaths we experience in this lifetime. This lifetime of cyclic rebirth brings us afflictions and suffering. Our mind is busy every day and at every moment. Constantly, we are trapped in a state of confusion, turidity, and confinement. Keep in mine that wandering thoughts have nothing to do with enlightenment ment; therefore, just let them rise and cease. In other words,
LET GO!

**Reply by shane0** [reply]
# 5. All experiences in meditation are illusory and insubstantial

As we overcome our constant wandering thoughts, we will experience a sense of serenity, clarity, and peace. We may also feel ineffable Dharma joy deep inside. Although this signifies our progress in meditation, do not cling to it or be too happy about it. Just maintain the pure knowing, and remember the teaching in The Diamond Sutra that "all appearances are illusory." Otherwise, we might give rise to something like pride-an obstacle, impeding our further achievement.
Furthermore, as a Chan saying goes, "When seeing a buddha, kill the buddha. When seeing a mara, kill the mara." Do not fall into the dichotomy of buddha and mara, good and evil, enlightenment and delusion, and the likes. Our original mind is the mind of the middle way. We meditate in accordance with the quality of our original mind to return to the true reality. When meditating on this very mind, just be in the present moment, aware and still.

**Reply by shane0** [reply]
# 6. Seek nothing and obtain nothing

The Platform Sutra states, "There is no cultivation or attain-ment; there is no defilement or impurity. Each and every being is innately enlightened, for we all have buddha nature and are originally awakened. Nothing can be established in the orig inally pure mind. Hence, there is no enlightenment to attain.
If we want to be awakened, or if we anticipate the moment of becoming enlightened, we are "defiling" our pure mind; these clouds of attachment and delusion prevent us from seeing the bright moon of our inherently true nature. Afflictions are ungraspable. Therefore, there is no need to look for enlightenment or purify our mind. Also, as The Vimalakirti Sutra teaches us, do not be attached to, or search for the Three Jewels. Just be in the present moment to reveal and realize our innate buddha nature. We need to let go of Dharma as well as non-Dharma.
This is a practice of sudden enlightenment.
It is quite a challenge to maintain pure awareness and constantly abide in the state of no-thought. If we feel sleepy, we may switch to the method of breath counting to awaken ourselves again. If our mind is scattered, we may observe our breath in order to bring our mind back. If our mind is clouded, we may contemplate on the impermanent, empty nature of phenomena.
If we feel bored or dull, we may look at the buddha statue and use his example of perfect compassion, wisdom, and vows to motivate us to be diligent. These are temporary antidotes to the problems we encounter in meditation. When our mind is awake and alert again, we can go back to the mind ground practice and naturally abide in our pure nature.
A passage in The Perfect Enlightenment Sutra summarizes the practice of mind ground: "At all times, do not give rise to deluded thoughts nor try to extinguish any deluded thought.  In a deluded state, do not discern it. Without discernment, make no differentiation either." This is the practice of following awareness, practicing in accord with perfect, luminous, pure awareness. It is a practice with no thoughts, conditions, deliberate efforts, attributes, or discriminations. This practice also resonates with the middle way reality. With persistent practice, eventually we will achieve the goal of perfect enlightenment: awakening the mind to see its true nature; seeing the true nature, one becomes a buddha.

**Reply by shane0** [reply]
When we see the red lights flashing, we know a train is approaching. When we see a train passing by right before our eyes, we know it is moving. When it has passed, and we no longer see the train in front of us, we know it has gone.
This analogizes the nature of our thoughts-rising, abiding, changing, and ceasing. No matter what type of train comes, how many passengers it has, how long it is, or how old it is, we do not care. Likewise, do not try to figure out the number of our thoughts, nor their qualities; just maintain our pure knowing, without any differentiation or judgment.
